### ISSUE TITLE
Commit and push button instead of push checkbox

### ISSUE THREAD

**Issue by dionisiydk** [pharo-vcs/iceberg#770]
It is feature request. 
I believe that previous Iceberg approach with two commit buttons was much much better than current one with extra checkbox to perform push after commit. 
Now I always forgot to push and it is super annoying. 

At least push checkbox should be placed closer to commit button and it should keep last chosen value.

**Comment by estebanlm** [comment]
no, this no. 
sorry, we tested with older version and it does not works.
what we can and should do is to remember last option.

**Comment by estebanlm** [comment]
there it is, I changed the title

**Comment by dionisiydk** [comment]
What I ask is not a new feature but different UI for existing one:
Now user selects checkbox and press commit - two mouse clicks on different panes.
I ask for the button which will "select checkbox and press commit" - one mouse click. 
Why it can't work? 

If you not agree then at least move checkbox to be nearly commit button.

**Comment by estebanlm** [comment]
Because it doesn't. I do not have a better answer that it is experience. 
So, you will press a checkbox, then tool will remember you did it. 
Then you will have just one button again.

**Comment by dionisiydk** [comment]
Maybe remembering is not necessary. What you think to select push checkbox by default?
Because it seems like a most common scenario.

**Comment by estebanlm** [comment]
nah, it is just your default, not common scenario (for example, is not the case for me).

**Comment by guillep** [comment]
The main problem is "Where to push". By default, when you clone a repository a branch does not have an upstream, and you have to set it manually in git. In iceberg we decided for the moment that the first repository to push to will become the upstream of your branch.

The thing is, until you don't push for the first time, Iceberg cannot know what is your remote to push.

Now, for the commit and push I'd like to have a kind of dropdown button like this

![](http://webuikits.com/wp-content/uploads/2012/10/Multipurpose-Dropdown-Button.jpg)

but we don't have that widget

**Comment by jecisc** [comment]
Another thing is that sometimes we don't want Iceberg to remember our choice.

For example, in general, I work and I can do multiple commits without pushing, then when a finish a feature or my work day I push. And I might push by checking the checkbox but later if I begin a new feature or a new work day I do not want to push by default.
